src/test/ui/target-feature/invalid-attribute.rs

```diff
@@ -24,12 +24,6 @@
 //~^ ERROR malformed `target_feature` attribute
 unsafe fn foo() {}
 
-#[target_feature(enable = "sse2")]
-//~^ ERROR `#[target_feature(..)]` can only be applied to `unsafe` functions
-//~| NOTE can only be applied to `unsafe` functions
-fn bar() {}
-//~^ NOTE not an `unsafe` function
-
```

Comment:
Why was this removed? It overlaps with the new feature-gate test now, but (at least in the current state of the PR) I see no reason to remove this, it's just unnecessary churn. Let's just leave it be, it doesn't do any harm.

Reply:
IIRC Centril asked me to remove it. I think both letting it and removing it makes sense, but I think I shouldn't remove it if the other test that exercises it only gets executed on certain platforms.

Reply:
This test only gets executed on x86 targets, too (see all the ignore-* lines at the top). There's just no target-independent way to test these attributes, since there's no target feature names that are recognized on all platforms.